### PR TITLE
awsume 4.5.1

### PR DESCRIPTION
**Pull request by dtrodrigues**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by dtrodrigues** [comment]
https://github.com/trek10inc/awsume/issues/152

**Review by chenrui333 — APPROVED** [review]

File: Formula/awsume.rb [hunk 1]
```diff
@@ -55,35 +55,35 @@ class Awsume < Formula
...
+    sha256 "e7b021f7241115872f92f43c6508082facffbd1c048e3c6e2bb9c2a157e28937"
   end
 
   def install
     virtualenv_install_with_resources
   end
 
   test do
-    assert_match version.to_s, shell_output("#{bin}/awsume -v 2>&1")
+    assert_match version.to_s, shell_output(". #{bin}/awsume -v 2>&1")
```
Comment (carlocab) [inline Formula/awsume.rb:82]: Can we wrap this in an exec script?

File: Formula/awsume.rb [hunk 2]
```diff
@@ -55,35 +55,35 @@ class Awsume < Formula
...
 
     file_path = File.expand_path("~/.awsume/config.yaml")
     shell_output(File.exist?(file_path))
```
Comment (carlocab) [inline Formula/awsume.rb:85]: There's probably an idiomatic way of doing this...

**Comment by dtrodrigues** [comment]
@carlocab  Not sure the best way of handling the alias, installation/usage instructions are in: https://github.com/trek10inc/awsume/blob/8410bd4ecac3b4865f3d6f72a7dbe20b5c04fa8f/docs/general/quickstart.md

**Comment by carlocab** [comment]
It says that installing via `pip` will try to create that alias, though. The equivalent for us is probably wrapping in an exec script.

**Pull request closed without merging**